src/app/phast/phast.service.ts

```diff
@@ -45,6 +45,7 @@ export class PhastService {
   stepTab: BehaviorSubject<StepTab>;
   specTab: BehaviorSubject<StepTab>;
   calcTab: BehaviorSubject<string>;
+  modalOpen: BehaviorSubject<boolean>;
```

Comment:
this BehaviorSubject should be in stackLossService. phast is "Process Heating" not realated.